en/index.json

```diff
@@ -1,5 +1,7 @@
 {
   "common": {
+    "run_on_background": "Run on Background",
+    "less_than_1000000": "Value should be  less than 1000000",
```

Comment:
Why do we need to hardcode the 1000000? Shouldn't we be doing a format string instead so we can change the number easily?

Reply:
Yup, use templating?